releng_sop/koji_clone_tag_for_release_milestone.py

```diff
@@ -0,0 +1,143 @@
+# -*- coding: utf-8 -*-
+"""Clone tag for release milestone.

```

Comment:
Put this on new line for better readability.
<nitpicking>I'd also put 2 empty lines before this docstring</nitpicking>